### ISSUE TITLE
Seal UC SRS

### ISSUE THREAD

**Issue by hlynch** [iceberg-project/ICEBERG-administration#10]
Hoping to finish all sections of the seal SRS, need details for software interface and communication interface.

**Comment by hlynch** [comment]
Understood about them being living documents but I want to send these out to PolarTEC and also they'll soon be linked to from the new website so I'm hoping that they look finished at least to a version 1.0 even if we know they aren't final.

**Comment by iparask** [comment]
@bspitzbart: can you check if the pushed version of the SRS is based on our last discussion?

As of now, I do not think we have any external communication or software requirements. As a result, these two sections are still empty.

**Comment by bspitzbart** [comment]
Yes, that looks good. Ver 1.0 and section 5 completed.  I will add section 5 to the other SRSs.
